util/certs/types.go

```diff
@@ -49,6 +49,11 @@ type Config struct {
 
 // NewSignedCert creates a signed certificate using the given CA certificate and key.
 func (cfg *Config) NewSignedCert(key *rsa.PrivateKey, caCert *x509.Certificate, caKey *rsa.PrivateKey) (*x509.Certificate, error) {
+	return cfg.NewSignedCertPKCS8(key, caCert, caKey)
```

Comment:
Same, why add a new function for this? Are you suggesting we deprecate `NewSignedCert`?

Reply:
+1 I think we can get away with updating `NewSignedCert` since it shouldn't break existing uses even if we are changing the signature.

Reply:
I agree in theory, but when I tried to do it that way, the go-apidiff job complained that I broke the api compatibility

Reply:
Hard to say without seeing the code that broke the job, but the signature here is exactly the same so I don't see why it would